nodejs/publish-layers.sh

```diff
@@ -5,77 +5,178 @@ set -Eeuo pipefail
 BUILD_DIR=nodejs
 BUCKET_PREFIX=nr-layers
 DIST_DIR=dist
-NJS12X_DIST=$DIST_DIR/nodejs12x.zip
-NJS14X_DIST=$DIST_DIR/nodejs14x.zip
+
+NJS12X_DIST_ARM64=$DIST_DIR/nodejs12x.arm64.zip
+NJS14X_DIST_ARM64=$DIST_DIR/nodejs14x.arm64.zip
+
+NJS12X_DIST_X86_64=$DIST_DIR/nodejs12x.x86_64.zip
+NJS14X_DIST_X86_64=$DIST_DIR/nodejs14x.x86_64.zip
 
 EXTENSION_DIST_DIR=extensions
-EXTENSION_DIST_URL=https://github.com/newrelic/newrelic-lambda-extension/releases/download/v2.0.4/newrelic-lambda-extension.zip
+EXTENSION_DIST_URL_ARM64=https://github.com/newrelic/newrelic-lambda-extension/releases/download/v2.0.5/newrelic-lambda-extension.arm64.zip
+EXTENSION_DIST_URL_X86_64=https://github.com/newrelic/newrelic-lambda-extension/releases/download/v2.0.5/newrelic-lambda-extension.x86_64.zip
 EXTENSION_DIST_ZIP=extension.zip
 EXTENSION_DIST_PREVIEW_FILE=preview-extensions-ggqizro707
 
-REGIONS=(
+# Regions that support arm64 architecture
+REGIONS_ARCH=(
   ap-northeast-1
-  ap-northeast-2
   ap-south-1
   ap-southeast-1
   ap-southeast-2
-  ca-central-1
   eu-central-1
-  eu-north-1
   eu-west-1
   eu-west-2
-  eu-west-3
-  sa-east-1
   us-east-1
   us-east-2
-  us-west-1
   us-west-2
 )
 
+# Regions that don't yet support arm64 architecture
+REGIONS_NO_ARCH=(
+  ap-northeast-2
+  ca-central-1
+  eu-north-1
+  eu-west-3
+  sa-east-1
+  us-west-1
+)
+
 function usage {
     echo "./publish-layers.sh [nodejs12.x|nodejs14.x]"
 }
 
-function download-extension {
+function download-extension-arm64 {
+    rm -rf $EXTENSION_DIST_DIR $EXTENSION_DIST_ZIP
+    curl -L $EXTENSION_DIST_URL_ARM64 -o $EXTENSION_DIST_ZIP
+    unzip $EXTENSION_DIST_ZIP -d .
+    rm -f $EXTENSION_DIST_ZIP
+}
+
+function download-extension-x86 {
     rm -rf $EXTENSION_DIST_DIR $EXTENSION_DIST_ZIP
-    curl -L $EXTENSION_DIST_URL -o $EXTENSION_DIST_ZIP
+    curl -L $EXTENSION_DIST_URL_X86_64 -o $EXTENSION_DIST_ZIP
     unzip $EXTENSION_DIST_ZIP -d .
     rm -f $EXTENSION_DIST_ZIP
 }
 
-function build-nodejs12x {
-    echo "Building new relic layer for nodejs12.x"
-    rm -rf $BUILD_DIR $NJS12X_DIST
+function build-nodejs12x-arm64 {
+echo "Building new relic layer for nodejs12.x (arm64)"
+    rm -rf $BUILD_DIR $NJS12X_DIST_ARM64
+    mkdir -p $DIST_DIR
+    npm install --prefix $BUILD_DIR newrelic@latest @newrelic/aws-sdk@latest
```

Comment:
The Node agent team's just removed the @newrelic/aws-sdk dependency as a separate install. It's included within the agent now.